ci_framework/roles/build_containers/README.md

```diff
@@ -0,0 +1,32 @@
+# build_containers
+Role to build openstack containers using tcib
+
+## Privilege escalation
+become - Required to install and execute tcib
+
+## Parameters
+
+* `cifmw_build_containers_tcib_src`: (String) Repository of tcib to be installed
+* `cifmw_build_containers_basedir`: (String) Directory where we will hold all tcib files. Default to `cifmw_basedir/artifacts/containers_build` which defaults to `~/ci-framework-data/`.
+* `cifmw_build_containers_authfile_path`: (String) Authorization file to push containers to registry. Default to `${XDG_RUNTIME_DIR}/containers/auth.json`.
+* `cifmw_build_containers_push_containers`: (Boolean) Whether push containers after build or not. Default to `false`.
+* `cifmw_build_containers_timestamper_cmd`: (String) Timestamp to be added in the logs
+* `cifmw_build_containers_container_name_prefix`: (String) Prefix to be added in the name of the containers. Default to `openstack`.
+* `cifmw_build_containers_buildah_push`: (Boolean) Whether push containers with buildah or not. Default to `false`.
```

Comment:
Can we keep only those parameters here which were actually required for building containers like
`cifmw_build_containers_push_registry`, `cifmw_build_containers_push_containers`. 

for example: [edpm_prepare](https://github.com/openstack-k8s-operators/ci-framework/tree/main/ci_framework/roles/edpm_prepare) and [edpm_deploy](https://github.com/openstack-k8s-operators/ci-framework/tree/main/ci_framework/roles/edpm_deploy)

Reply:
Although we are not using it, they are parameters that is being passed in the build_containers.sh.j2 template and options that can be overwritten.